### PR TITLE
delete your bot

### PR DESCRIPTION
**Pull request by deagahelio**

### CODE REVIEW

**Comment by matcool** [comment]
I agree with the change.
Reasons:
I thought pizzabot would change my lyfe, and my experience with discord. I was having fun wit it until it started sending pornographic content. And now after seing the sorce cold, it just made Me hate it more. please delete.

**Review by matcool — APPROVED** [review]
I do not like it.

File: pizzabot.js [hunk 1]
```diff
@@ -1,74 +0,0 @@
...
-					finalmessage += " " + currentword;
-				}
-			}
-		}
-		fs.appendFile(msg.channel.id + '.txt', message.replace(/\n/g, " ") + "\n", function (err) {});
-		msg.channel.send(finalmessage);
-	}
-});
-
-client.login('BOT_TOKEN');
```
Comment (matcool) [inline pizzabot.js:74]: Do not mean what "TOKEN" is, got an error everytime.

**Comment by deagahelio** [comment]
i do not make prs for them to be closed 😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡

**Pull request closed without merging**